resources/packs/gcp/gcp.lr

```diff
@@ -172,6 +172,44 @@ private gcp.project.computeService {
   zones() []gcp.project.computeService.zone
   // List of backend services
   backendServices() []gcp.project.computeService.backendService
+  // List of IP addresses
+  addresses() []gcp.project.computeService.address
+}
+
+// GCP Compute Address
+private gcp.project.computeService.address @defaults("name address addressType") {
+  // Unique identifier
+  id string
+  // Static IP address
+  address string
+  // Address type
+  addressType string
+  // Creation timestamp
+  created time
+  // Address description
+  description string
+  // IP version
+  ipVersion string
+  // Endpoint type
+  ipv6EndpointType string
+  // Address name
+  name string
+  // URL of the network in which to reserve the address
+  networkUrl string
```

Comment:
I would expect that `.network()` returns the nextwork resource

Reply:
I have not added these intentionally for now since we don't have the use-case for it. It's missing on a lot of the resources, not just on that one

Reply:
I do not understand why it is complicated to add this. This should be just five lines since all we need to init the resource is in the url

Reply:
It was not a straight-forward implementation. There were other open issues that needed to be addressed first. Here's the commit that fixes that https://github.com/mondoohq/cnquery/pull/934/commits/abcce78c7ae8d90934c103240733ef76ecbb18c8